sycl/test/fpga_tests/fpga_aocx.cpp

```diff
@@ -9,7 +9,9 @@
 // REQUIRES: aoc, accelerator
 
 /// E2E test for AOCX creation/use/run for FPGA
-// Produce an archive with device (AOCX) image
+// Produce an archive with device (AOCX) image. Avoid appending objects to
+// leftover archives.
+// RUN: if [ -f %t_image.a ]; then rm %t_image.a; fi
```

Comment:
I think this requires to put requirement for Linux as it's `bash`/`sh` syntax. I'm not sure if `llvm-lit` will parse it.

Reply:
You're right, this didn't get parsed by the CI's LIT tool. I've submitted a "less bashy" variant.